src/lib/Twig/Components/Badge.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\DesignSystemTwig\Twig\Components;
+
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+use Symfony\UX\TwigComponent\Attribute\PreMount;
+
+#[AsTwigComponent]
+final class Badge
+{
+    private const MAX_BADGE_VALUE = 99;
```

Comment:
A little in reference to my second comment, I think it would be good idea to expose this value in twig and set it to data-attribute and use it in JS file. Thanks to it this value would be defined only in one place instead of two (easier to update)